### DISCUSSION TITLE
Reaching a concensus on value name conventions

### DISCUSSION THREAD

**Discussion by tliron** [oasis-tcs/tosca-specs#230] [Ideas]
As you may recall, I proposed that value names use dash case as a subtle differentiator from keynames, which use snake case (underscores). This includes properties, attributes, inputs, operations, capabilities, relationships, etc.

I've done this consistently in my own work and I it has helped mark these as "user defined", both in reading and also while authoring TOSCA.

Hopefully in the future we will have a TOSCA colorizer that could present these as a different color in an IDE.

I'm willing to fix the spec accordingly, but obviously we need consensus.

**Comment by lauwers** [comment]
I'm fine with doing this.

**Reply by tliron** [reply]
I will take a risk and treat this as consensus.

Am working on getting a PR out tonight.
